src/bug-filing/common/get-issue-details-html.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { escape } from 'lodash';
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { title } from '../../content/strings/application';
+import { IssueUrlCreationUtils } from './issue-filing-url-string-utils';
+
+const buildBodySection = (headingHtml: string, contentHtml: string): string => {
+    return `<div><b>${headingHtml}</b></div><div>${contentHtml}</div><br>`;
+};
+
+const buildIssueDetails = (help: string, helpUrl: string, ruleId: string): string => {
```

Comment:
I'd be tempted to call this `buildIssueDetailsHtml` as this makes it consistent with the other methods that return HTML that's ready to be used (untrusted data escaped).

(Also exists in the old code)

Reply:
updated.